src/main/java/hudson/plugins/jobConfigHistory/FileHistoryDao.java

```diff
@@ -25,25 +25,13 @@
 
 import static java.util.logging.Level.FINEST;
 
-import java.io.BufferedOutputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.PrintStream;
+import java.io.*;
 import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Calendar;
-import java.util.Collections;
-import java.util.Date;
-import java.util.GregorianCalendar;
-import java.util.SortedMap;
-import java.util.TreeMap;
+import java.util.*;

```

Comment:
Replace `java.util.*` by the actual required class.

Reply:
Done.